pandas/util/mixins.py

```diff
@@ -0,0 +1,28 @@
+"""
+Mixins which can't go to core.common or core.base due to circular imports
+"""
```

Comment:
@kawochen This doesn't help (just tested with "Some" instead of moving the "NoNewAttributesMixin" there):

```
Traceback (most recent call last):
  File "C:\Program Files (x86)\JetBrains\PyCharm Community Edition 143.165\helpers\pycharm\utrunner.py", line 121, in <module>
    modules = [loadSource(a[0])]
  File "C:\Program Files (x86)\JetBrains\PyCharm Community Edition 143.165\helpers\pycharm\utrunner.py", line 41, in loadSource
    module = imp.load_source(moduleName, fileName)
  File "C:\data\external\pandas\pandas\tests\test_series.py", line 20, in <module>
    import pandas as pd
  File "C:\data\external\pandas\pandas\__init__.py", line 42, in <module>
    import pandas.core.config_init
  File "C:\data\external\pandas\pandas\core\config_init.py", line 17, in <module>
    from pandas.core.format import detect_console_encoding
  File "C:\data\external\pandas\pandas\core\format.py", line 8, in <module>
    from pandas.core.base import PandasObject
  File "C:\data\external\pandas\pandas\core\base.py", line 10, in <module>
    import pandas.core.strings as strings
  File "C:\data\external\pandas\pandas\core\strings.py", line 7, in <module>
    from pandas.core.base import Some
ImportError: cannot import name Some
```